flank-scripts/src/test/kotlin/flank/common/FilesTest.kt

```diff
@@ -10,6 +11,7 @@ internal class FilesTest {
 
     @Test
     fun `Should create symbolic file at desired location`() {
+        Assume.assumeFalse(isWindows)
```

Comment:
does it fail on windows? I assume it should work cause under the hood it uses java api

Reply:
It does. https://github.com/Flank/flank/issues/1434

Reply:
oh I see, I tested it on CI, so it does not have problem with permissions